utils/challenges/challenges.js

```diff
@@ -0,0 +1,36 @@
+import firebase from "../firebase/firebase";
```

Comment:
If we plan on adding more functions to this file, would it be a good idea to move it under `utils/db`?

Reply:
Will do, I think this is not in master yet so I missed it but that makes sense.